### PR TITLE
Game over at computer page #198

### LINKED ISSUE #198: Game over at computer page
After last changes the game over event at computer page doesn't function anymore.

### PR DESCRIPTION
**Pull request by dzianisshupenka**
Fixed, now it works.
closes #198

### CODE REVIEW

File: frontend/src/components/Chessboard.vue [hunk 1]
```diff
@@ -198,6 +202,19 @@ export default {
...
         lastMove: lastMove
       });
+      if (ply != this.game.history().length) {
+        this.board.set({
+          movable: {
+            color: null
+          }
+        });
+      } else {
+        this.board.set({
```
Comment (juggler39) [inline frontend/src/components/Chessboard.vue]: This piece of code doesn't work as supposed. Please remove it.

**Review by juggler39 — CHANGES_REQUESTED** [review]
Please remove that marked code.

File: frontend/src/components/Chessboard.vue (hunk 1 above)
Comment (dzianisshupenka) [reply frontend/src/components/Chessboard.vue]: Removed

**Review by juggler39 — APPROVED** [review]
Good

**Pull request merged**